distributed/deploy/tests/test_spec_cluster.py

```diff
@@ -131,6 +132,33 @@ class MyCluster(SpecCluster):
         assert len(cluster.workers) == 2
 
 
+@pytest.mark.asyncio
+async def test_adaptive_killed_worker(cleanup):
+    with dask.config.set({"distributed.deploy.lost-worker-timeout": 1}):
+
+        with SpecCluster(
+            worker={"cls": Nanny, "options": {"nthreads": 1}},
+            scheduler={"cls": Scheduler, "options": {"port": 0}},
+        ) as cluster:
+
+            with Client(cluster) as client:
```

Comment:
If you are doing an async style test (which I recommend) then I encourage you to use an asynchronous cluster and client using `async with` and the `asynchronous=` keyword.  Other tests in this file should provide an example.

Reply:
Ah that's good to know about - I've updated the tests to include the `async with`!